Resources/Prototypes/DeltaV/Entities/Objects/Devices/Syndicate_Gadgets/reinforcement_teleporter.yml

```diff
@@ -18,3 +18,20 @@
     sound: /Audio/Animals/mouse_squeak.ogg
   - type: UseDelay
     delay: 300
+
+- type: entity
+  parent: ReinforcementRadioSyndicate
+  id: ReinforcementRadioSyndicateParamedic
+  name: syndicate paramedic reinforcement radio
+  description: Call in a syndicate paramedic instantly! Medical License not included.
```

Comment:
Let's get jokes out of the description, and keep it purely useful/loreful.

Reply:
1984

Reply:
jokes like that are a big part of the charm in 13/14, since anybody can contribute people put a little piece of themselves in their work, removing that just makes the game bland

Reply:
uplink description should be serious description should be funi